packages/cli/internal/pkg/aws/cdk/deploy_app.go

```diff
@@ -11,6 +11,15 @@ var mkDirTemp = os.MkdirTemp
 
 func (client Client) DeployApp(appDir string, context []string, executionName string) (ProgressStream, error) {
 	tmpDir, _ := mkDirTemp(appDir, "cdk-output")
+	// The temp directory defaults to 700 permissions, which can prevent Docker
+	// from mounting directories under it, which can prevent the CDK's bundling
+	// process from working, with errors like:
+	// docker: Error response from daemon: error while creating mount source path '...': mkdir ...: permission denied.
+	// So we need to change the permissions
```

Comment:
@nbraid Has the Docker build process I am talking about here been removed? This is a bit unrelated to the point of this PR, and I'd like to remove it, but I needed it when I added it for running the CDK to actually work (along with needing the MFA implementation).

Reply:
Docker has been removed in the current build